unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
From that link, under the unix port: "every byte is still accounted for". That doesn't seem to apply here, since as I described I can remove quite a few lines and build still succeeds. The more important question would be *how that affects the final binary in terms of functionality*. If a single line is so important, then this file needs to be revised to remove all other unnecessary lines.